verilog/analysis/verilog_linter_test.cc

```diff
@@ -383,25 +383,69 @@ TEST_F(VerilogLinterTest, MultiByteUTF8CharactersAreOnlyCountedOnce) {
   EXPECT_EQ(diagnostics.second, "");
 }
 
+TEST(VerilogLinterDocumentationTest, ParseCustomCitations) {
+  absl::string_view citations_raw =
+      "struct-union-name-style:one line citation is ok\n"
+      "signal-name-style:multi line citation\\\n"
```

Comment:
More test suggestions (and possibly code adaption needs)
  - what if there is a space before and/or after the colon: will it be properly stripped ?
  - what if we have \r\n as newlines (verible also runs on windows, so we need to expect that)